verticapy/tests_new/performance/vertica/collection/test_export_simple.py

```diff
@@ -0,0 +1,194 @@
+"""
+Copyright  (c)  2018-2024 Open Text  or  one  of its
+affiliates.  Licensed  under  the   Apache  License,
+Version 2.0 (the  "License"); You  may  not use this
+file except in compliance with the License.
+
+You may obtain a copy of the License at:
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless  required  by applicable  law or  agreed to in
+writing, software  distributed  under the  License is
+distributed on an  "AS IS" BASIS,  WITHOUT WARRANTIES
+OR CONDITIONS OF ANY KIND, either express or implied.
+See the  License for the specific  language governing
+permissions and limitations under the License.
+"""
+import logging
+import os
+import tarfile
+
+import pytest
+
+from verticapy.performance.vertica import QueryProfiler
+from verticapy.performance.vertica.collection.profile_import import (
+    ProfileImport,
+)
+from verticapy._utils._sql._sys import _executeSQL
+
+
+class TestQueryProfilerSimple:
+    """
+    Contains tests related to creating bundles of parquet data
+    """
+
+    def test_profile_export(self, amazon_vd, schema_loader, tmp_path):
```

Comment:
doc string is missing in all test functions.

Reply:
added docstring.